src/PKSim.Infrastructure/Reporting/Markdown/Builders/BuildingBlockMarkdownBuilder.cs

```diff
@@ -0,0 +1,25 @@
+﻿using PKSim.Core.Model;
+using PKSim.Infrastructure.Reporting.Markdown.Extensions;
+
+namespace PKSim.Infrastructure.Reporting.Markdown.Builders
+{
+   public abstract class BuildingBlockMarkdownBuilder<T> : MarkdownBuilder<T> where T : PKSimBuildingBlock
```

Comment:
We always end up with an abstract class for BuildingBlock. Might as well use it!